brainstormforce/ultimate-addons-for-gutenberg
# Enhancement: Spectra Asset Regeneration Implemented for Button Inherit from Theme Setting Updates.

### Description
Enhancement: Spectra Asset Regeneration Implemented for Button Inherit from Theme Setting Updates.

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### How has this been tested?
 - Check Regenerate Assets functionality.
 - Check Button Inherit from theme functionality.

### Checklist:
